zbarcam/zbarcam.py

```diff
@@ -91,13 +91,7 @@ def _on_texture(self, instance):
     def _detect_qrcode_frame(cls, texture, code_types):
         image_data = texture.pixels
         size = texture.size
-        fmt = texture.colorfmt.upper()
-        # PIL doesn't support BGRA but IOS uses BGRA for the camera
-        # if BGRA is detected it will switch to RGBA, color will be off
-        # but we don't care as it's just looking for barcodes
-        if cls.is_ios() and fmt == 'BGRA':
-            fmt = 'RGBA'
-        pil_image = PIL.Image.frombytes(mode=fmt, size=size, data=image_data)
+        pil_image = PIL.Image.frombytes(mode='RGBA', size=size, data=image_data)
```

Comment:
Minor: Flake8 is complaining, this line is one character too long :grimacing: 
```
zbarcam/zbarcam.py:94:80: E501 line too long (80 > 79 characters)
```

Reply:
Also would you mind putting a code comment and linking to #41 explaining why we're doing forcing the mode to RGBA.
That would save people time when going through the code later on